etc/default/systemd-status-mail

```diff
@@ -14,6 +14,12 @@ ADDRESS=
 # RELAYHOST specifies the mail relay used by mailx if specified
 #RELAYHOST=relay.example.com
 
+# If the RELAYHOST requires authentication set the username here
+#MAILX_AUTH_USER=
+
+# If the RELAYHOST requires authentication set the password here
```

Comment:
Please document, that the file must be readable by the "systemd-status-mail" user and if a password is set, it should be only readable by this user. So root:root/0600 will not work.

Reply:
Comment added, see the recent commit for my suggestion.